CHANGELOG.md

```diff
@@ -20,6 +20,11 @@ For older changes see the [archived Singularity change log](https://github.com/a
   regression introduced in 1.1.4.
 - Avoid incorrect error when requesting fakeroot network.
 
+### Backported from main tree
+
+- Build via zypper on SLE systems will use repositories of host via
+  suseconnect-container
+
```

Comment:
Please just include this in the list of "Bug fixes".  All of the bug fixes have been backported from the main branch.